repo/buildrepo/build.go

```diff
@@ -129,7 +131,13 @@ func NewCAFSStore(ctx context.Context, cfg *config.Config) (store cafs.Filestore
 			},
 			ipfs.OptsFromMap(cfg.Store.Options),
 		}
-		return ipfs.NewFilestore(fsOpts...)
+		store, err = ipfs.NewFilestore(fsOpts...)
```

Comment:
Won't let me comment on the actual import, but the change in cmd from importing `qfs/cafs/ipfs` as `qipfs` makes understanding what's happening much easier. 
Recommend changing it in `repo/buildrepo/build.go` as well! Perhaps also `qipfs_http`